app/controllers/search_controller.rb

```diff
@@ -10,19 +10,15 @@ def explore
 
   def search
   	sort_arr = sort_order
-  	repositories_by_attributes = Repository.where("lower(title) LIKE ?
+  	@repositories = Repository.where("lower(title) LIKE ?
                                                 OR lower(description) LIKE ?
                                                 OR lower(user_username) LIKE ?
                                                 OR lower(category) LIKE ?",
                                   "%#{params[:q].downcase}%",
                                   "%#{params[:q].downcase}%",
                                   "%#{params[:q].downcase}%",
                                   "%#{params[:q].downcase}%")
-    repositories_by_categories = []
-    Category.where('lower(name) LIKE ?', params[:q].downcase).each do |cat|
-      repositories_by_categories << cat.repository
-	  end
-    @repositories = repositories_by_attributes + repositories_by_categories
+
     @repositories = @repositories.uniq
```

Comment:
so i updated this and made it shorted but i'm not sure if the 'stuff' inside where() can become shorter too